### PR TITLE
[DS-1338] Address Display - Award Page Recipient & PoP

### PR DESCRIPTION
**Pull request by ejgullo**
- Reformatting the address display in recipient & pop fields for individual award pages
- Met the A/C, PM is aware that some values may look odd because we are missing some data, acceptable for the time being.

### CODE REVIEW

File: src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx [hunk 1]
```diff
@@ -61,24 +61,13 @@ export default class FinancialAssistanceDetails extends React.Component {
 
         // Location
         let popPlace = "Not Available";
-        let popZip = null;
-        if (award.pop_zip) {
-            popZip = award.pop_zip;
+        if (award.pop_country_code === 'USA') {
+            popPlace = `${award.pop_city}, ${award.pop_state_province} ${award.pop_zip}`;
```
Comment (kevinli-work) [inline src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx]: Why is city but no state/state but no city not being handled in this view?

File: src/js/components/award/contract/ContractDetails.jsx [hunk 2]
```diff
@@ -74,24 +74,25 @@ export default class ContractDetails extends React.Component {
...
-            popPlace = `${award.pop_city} ${popZip}`;
+        else if (stateProvince) {
+            cityState = stateProvince;
         }
-        else if (award.pop_city && !award.pop_state_province && !popZip) {
-            popPlace = award.pop_city;
+        if (award.pop_country_code === 'USA') {
+            popPlace = `${cityState} ${award.pop_zip}`;
+            popPlace +=
+            `\nCongressional District: ${award.pop_state_code}-${award.pop_congressional_district}`;
```
Comment (kevinli-work) [inline src/js/components/award/contract/ContractDetails.jsx]: What's the expected behavior when state code and/or congressional district isn't available?

File: src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx [hunk 3]
```diff
@@ -61,24 +61,13 @@ export default class FinancialAssistanceDetails extends React.Component {
...
+            popPlace +=
+            `\nCongressional District: ${award.pop_state_code}-${award.pop_congressional_district}`;
```
Comment (kevinli-work) [inline src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx]: Same, what if there is no state code or congressional district (see `award/603`)?

File: src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx (hunk 1 above)
Comment (ejgullo) [inline src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx]: forgot to update this file with this morning's new requests - fixing now

File: src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx (hunk 3 above)
Comment (ejgullo) [inline src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx]: same as above

File: src/js/components/award/contract/ContractDetails.jsx (hunk 2 above)
Comment (ejgullo) [inline src/js/components/award/contract/ContractDetails.jsx]: Confirmed with kaitlin, we will be hiding it.  I'll add this logic in.

**Comment by geoffgeiling** [comment]
Design review conducted and PR approved

**Pull request merged**